data/changelogs/kv.yaml

```
@@ -0,0 +1,15 @@
+---
+link: "/kv/platform/changelog/"
+productName: Workers KV
+productLink: "/kv/"
+productArea: Developer platform
+productAreaLink: /workers/platform/changelog/platform/
+entries:
+  - publish_date: "2024-08-08"
+    title: New KV Analytics API
+    description: |-
+      Workers KV now has a new [analytics API](/kv/observability/metrics-analytics/) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics on both operations and storage, including latency metrics for both read and write operations to Workers KV.
+
+      The metrics queried by this API will power upcoming improvements to the Workers KV dashboard analytics.
+
+      The legacy Workers KV [analytics REST API](/api/operations/workers-kv-request-analytics-query-request-analytics) will be turned off as of January 31st, 2025. Developers using this API will receive a series of email notifications prior to the shutdown of the legacy API.
```

```suggestion
  - publish_date: "2024-08-08"
    title: New KV Analytics API
    description: |-
      Workers KV now has a new [metrics dashboard](/kv/observability/metrics-analytics/#view-metrics-in-the-dashboard) and [analytics API](/kv/observability/metrics-analytics/#query-via-the-graphql-api) that leverages the [GraphQL Analytics API](/analytics/graphql-api/) used by many other Cloudflare products. The new analytics API provides per-account and per-namespace metrics for both operations and storage, including latency metrics for read and write operations to Workers KV.

      The legacy Workers KV [analytics REST API](/api/operations/workers-kv-request-analytics-query-request-analytics) will be turned off as of January 31st, 2025. Developers using this API will receive a series of email notifications prior to the shutdown of the legacy API.
```

@elithrar was thinking of only merging when dashboard is complete to minimize unnecessary back & forth given this is probably 1-2 weeks away.